skyvern/client/client.py

```
@@ -3926,6 +3947,14 @@ async def main() -> None:
         )
         return _response.data
 
+    @property
+    def workflows(self):
+        if self._workflows is None:
+            from .workflows.client import AsyncWorkflowsClient  # noqa: E402
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)